tests/test_collection.py

```diff
@@ -67,17 +67,21 @@ def make_col(base, name):
         self.assertRaises(errors.InvalidName, make_col, self.db.test, "test.")
         self.assertRaises(errors.InvalidName, make_col, self.db.test, "tes..t")
         self.assertRaises(errors.InvalidName, make_col, self.db.test, "tes\x00t")
-        self.assertFailure(self.coll.save("test"), TypeError)
-        self.assertFailure(self.coll.filemd5("test"), ValueError)
-        self.assertFailure(self.db.test.find(spec="test"), TypeError)
-        self.assertFailure(self.db.test.find(fields="test"), TypeError)
-        self.assertFailure(self.db.test.find(skip="test"), TypeError)
-        self.assertFailure(self.db.test.find(limit="test"), TypeError)
-        self.assertFailure(self.db.test.insert([1]), TypeError)
-        self.assertFailure(self.db.test.insert(1), TypeError)
-        self.assertFailure(self.db.test.update(1, 1), TypeError)
-        self.assertFailure(self.db.test.update({}, 1), TypeError)
-        self.assertFailure(self.db.test.update({}, {}, 'a'), TypeError)
+        self.assertRaises(TypeError, self.coll.save, "test")
+        self.assertRaises(ValueError, self.coll.filemd5, "test")
+        #self.assertFailure(self.db.test.find(spec="test"), TypeError)
+        #self.assertFailure(self.db.test.find(fields="test"), TypeError)
+        #self.assertFailure(self.db.test.find(skip="test"), TypeError)
+        #self.assertFailure(self.db.test.find(limit="test"), TypeError)
```

Comment:
They are replaced by `assertRaises` below. Commented ones should be removed, just forgot it.